membrane_curvature/tests/test_mdakit_membcurv.py

```diff
@@ -174,10 +268,23 @@ def universe():
 
 
 @pytest.fixture()
-def mdtraj_po4():
+def leaflets():
+    return ["lower", "upper"]
+
+
+@pytest.fixture()
+def topol():
+    top = {'small': md.load(GRO_PO4_SMALL).topology,
+           'all': md.load(GRO_PO4).topology}
```

Comment:
This on the other hand is a good use of a fixture. If one test modifies the GRO_PO4 MDTraj object, we don't want that to affect another test.